addons/libs/images.lua

```diff
@@ -0,0 +1,344 @@
+--[[
+    A library to facilitate image primitive creation and manipulation.
+]]
+
+local images = {}
+local meta = {}
+
+windower.prim.saved_images = {}
+local dragged
```

Comment:
You can disable dragging on a per-addon basis. There is a `draggable` attribute that can be passed to it as `true` or `false`, which defaults to `true`:
```lua
text = texts.new('Some text', { draggable = false })
```

The TParty addon does this, for example, because it explicitly positions the text boxes. If you copied the API from the `texts.lua`, this functionality would be included in the `images.lua` library, so you could make *BarFiller* use the custom X and Y coordinates and disable it, but other addons could still use it.